docker/solr/schema.xml

```diff
@@ -1123,8 +1118,16 @@
     <field name="country" type="string" indexed="true" stored="true" required="false" multiValued="false" />
     <field name="text" type="text_general" indexed="true" stored="false" required="false" multiValued="false" />
 
+    <!-- all the languages (we currently support) of OCR, mapped to the various
+         built-in language-specific field types -->
+    <field name="ocr_eng" type="text_en" indexed="true" stored="true" required="false" multiValued="false" />
+    <field name="ocr_fre" type="text_fr" indexed="true" stored="true" required="false" multiValued="false" />
+    <field name="ocr_spa" type="text_es" indexed="true" stored="true" required="false" multiValued="false" />
+    <field name="ocr_ita" type="text_it" indexed="true" stored="true" required="false" multiValued="false" />
+    <field name="ocr_ger" type="text_de" indexed="true" stored="true" required="false" multiValued="false" />
+    <copyField source="ocr_*" dest="ocr" />
```

Comment:
I considered this problem actually.  It's worse than you may think, though.  The language codes ONI (chronam) uses are not compatible with what a managed schema uses.  For instance, "spa" rather than "es" and "ger" rather than "de".  We'd need to find (or build) some kind of lookup even if we had a managed schema.